README: app needs to be loaded before starting the server and moved the section towards the top (#124)
fukamachi/caveman: README.markdown

;   writing /path/to/myapp/templates/layout/default.tmpl
```

### Start a server

> This is a example that assumes that the name of your application is "myapp".

Reviewer: ```suggestion
This is an example that assumes that the name of your application is "myapp".
```

Before starting the server, you must first load your app.

```common-lisp
(ql:quickload :myapp)
```

Your application has functions named `start` and `stop` to start/stop your web application. 

```common-lisp
(myapp:start :port 8080)
```

> As Caveman is based on Clack/Lack, you can choose which server to run on -- Hunchentoot, mod_lisp or FastCGI.

Reviewer: Changing the list of example web servers.
This is an original text, but I'd like to change at this good chance.

```suggestion
As Caveman is based on Clack/Lack, you can choose which server to run on -- Hunchentoot, Woo or Wookie, etc.
```

```common-lisp
(myapp:start :server :hunchentoot :port 8080)
(myapp:start :server :fcgi :port 8080)
```

> I recommend you use Hunchentoot on a local machine, and use FastCGI/Woo in a production environment.

Reviewer: FastCGI handler is obsolete.

```suggestion
I recommend you use Hunchentoot on a local machine, and use Woo in a production environment.
```
